frontend/src/app/modules/workspace/sidebar/sidebar.component.html

```diff
@@ -1,38 +1,60 @@
 <mat-nav-list>
-    <a class="logo" routerLink="">
-        <img src="../../../../assets/icons/logo-white.svg" alt="logo" />
-    </a>
+  <a class="logo" routerLink="">
+    <img src="../../../../assets/icons/logo-white.svg" alt="logo" />
+  </a>
 
-    <div class="workspace-container">
-        <div class="workspace-user">
-            <span class="user-subheader">
-                {{this.user.workspaceName}}
-            </span>
-            <span class="member-subheader">
-                1 member
-            </span>
-        </div>
-        <button mat-button class="workspace-button" routerLink="">
-            Invite teammates
-        </button>
+  <div class="workspace-container">
+    <div class="workspace-user">
+      <mat-form-field class="teams-box" appearance="fill">
+        <mat-label>Team:</mat-label>
+        <mat-select
+          style="color: red"
+          class="select-cls"
+          [(ngModel)]="value"
+          (selectionChange)="onChange()"
+        >
+          <mat-option
+            class="opt"
+            *ngFor="let team of this.user.teams"
+            [value]="team.name"
+          >
+            {{ team.name }}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+      <!-- </mat-form-field> -->
+      <span class="member-subheader">Members: {{ this.members }}</span>
     </div>
-
-    <p matSubheader>Libraries</p>
-    
-    <button mat-list-item routerLink="" routerLinkActive="button-active">
-        <mat-icon svgIcon="personal"></mat-icon>
-        <span>Personal</span>
-    </button>
-    <button mat-list-item routerLink="shared-with-me" routerLinkActive="button-active">
-        <mat-icon svgIcon="share"></mat-icon>
-        <span>Shared with me</span>
-    </button>
-    <button mat-list-item routerLink="team" routerLinkActive="button-active">
-        <mat-icon svgIcon="team"></mat-icon>
-        <span>Team</span>
-    </button>
-    <button mat-list-item routerLink="settings" routerLinkActive="button-active">
-        <mat-icon svgIcon="settings"></mat-icon>
-        <span>Settings</span>
+    <button
+      (click)="showInvite()"
```

Comment:
If I'm not mistaken, we have an email service. It would be nice to send an email with an invitation instead of just showing a box